Makefile

```diff
@@ -1,11 +1,5 @@
-ifeq ($(OS), Windows_NT)
-	devnull := nul
-else
-	devnull := /dev/null
-endif
-
 next_version := 1.16.0
-tag := $(shell git describe --exact-match --tags 2>$(devnull))
+tag := $(shell git describe --exact-match --tags 2>git_describe_error.tmp; rm -f git_describe_error.tmp)
```

Comment:
this should not be changed in your PR.

Reply:
@wowi42: Rebasing on the latest master should remove this I think.